test/verify/check-services

```diff
@@ -397,16 +407,17 @@ WantedBy=default.target
         m.execute("systemctl start hourly_timer.timer")
         b.wait_present(svc_sel('hourly_timer.timer'))
         b.wait_in_text(svc_sel('hourly_timer.timer'), "Hourly timer")
+        wait_systemctl_timer("Fri 2020-04-10 03:05")
         self.assertIn("Fri 2020-04-10 03:05", m.execute("LC_ALL=C systemctl list-timers"))
         m.execute("timedatectl set-time '2020-04-10 03:07:00'")
-        b.wait_present(svc_sel('hourly_timer.timer'))
+        wait_systemctl_timer("Fri 2020-04-10 03:26")
         self.assertIn("Fri 2020-04-10 03:26", m.execute("LC_ALL=C systemctl list-timers"))
         m.execute("timedatectl set-time '2020-04-10 04:00:00'")
-        b.wait_present(svc_sel('hourly_timer.timer'))
+        wait_systemctl_timer("Fri 2020-04-10 04:05")
         # checks if timer runs on next hour at 5 min and 26 min
         self.assertIn("Fri 2020-04-10 04:05", m.execute("LC_ALL=C systemctl list-timers"))
         m.execute("timedatectl set-time '2020-04-10 04:10:00'")
-        b.wait_present(svc_sel('hourly_timer.timer'))
+        wait_systemctl_timer("Fri 2020-04-10 04:26")
```

Comment:
_b.wait_present(svc_sel('hourly_timer.timer'))_ was removed because I thought they would solve the race issue here. Since they didn't, I'm removing all such checks we had.

Reason why _b.wait_present(svc_sel('hourly_timer.timer'))_ could solve race was based on the idea that cockpit timer page refreshes after changing time using timedatectl.